### DISCUSSION TITLE
If possible, I hope I could use EarTrumpet on Win8 or Win7

### DISCUSSION THREAD

**Discussion by KahinHe** [File-New-Project/EarTrumpet#1174] [Completed ideas]
If possible, I hope I could use EarTrumpet on Win8 or Win7.
3Q~

**Comment by Cmdr-Keene** [comment]
Windows 7 is no longer supported by Microsoft and is really dangerous to use, lacks the programming APIs for modern development, and is not targeted by developers. Even Windows 8 is now a decade old and will lose support in a few months.

**Comment by riverar** [comment]
Windows 10+ is required for EarTrumpet to function. It can't be backported to unsupported OS versions, sorry.
